src/api/config.ts

```diff
@@ -0,0 +1,10 @@
+import axios from "axios";
+
+const instance = axios.create({
+  baseURL: "https://api.thecatapi.com/v1",
+});
+
+instance.defaults.headers["x-api-key"] =
```

Comment:
Are there any cons in this approach? And if there are any what would you change in order to consider your application production-ready?

Reply:
It is absolutely a bad practice. I wanted to let you just run the app without any configuration. For the production app I would use the `.env` file.

```
// .env file
REACT_APP_API_KEY=CONTENT_OF_THE_API_KEY

// config.ts file
instance.defaults.headers["x-api-key"] =process.env.REACT_APP_API_KEY